api/src/main/java/io/druid/data/input/impl/PrefetchableTextFilesFirehoseFactory.java

```diff
@@ -322,8 +317,10 @@ public LineIterator next()
                   openedObject.objectStream
               );
 
+              numRemainingObjects--;
+
               return new ResourceCloseableLineIterator(
-                  new InputStreamReader(stream, Charsets.UTF_8),
```

Comment:
Could you please add forbidden-apis rule about `Charsets`?

Reply:
There are quite lots of usages, so I raised #4872.